packages/fxa-graphql-api/src/gql/dto/input/consume-recovery-code.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class ConsumeRecoveryCodeInput {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
+
+  @Field({ description: 'The backup authentication (recovery) code to check' })
```

Comment:
I liked how comments else where specified this was a backup 2FA authentication code.

Reply:
Agree, I'll add this and also a description for the payload.